src/client/table_client.rs

```diff
@@ -2374,3 +2414,181 @@ impl Default for Builder {
         Self::new()
     }
 }
+
+pub struct ObTableAggregation {
+    table_query: ObTableClientQueryImpl,
+    /// this message is used to record the aggregation order and the corresponding aggregation name
+    message: Vec<String>,
```

Comment:
I guessed we should replace it with a better name.

Reply:
change to aggregation_operations